juniper_official/Subscriber-Service/pppoe-error-statistics.rule

```diff
@@ -0,0 +1,255 @@
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/ac-error-received;
+                }
+                type integer;
+                description "Number of ac-error-received";
+            }
+            field ac-error-sent {
+                sensor pppoe-error-statistics {
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/ac-error-sent;
+                }
+                type integer;
+                description "Number of ac-error-sent";
+            }
+            field generic-error-received {
+                sensor pppoe-error-statistics {
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/generic-error-received;
+                }
+                type integer;
+                description "Number of generic-error-received";
+            }
+            field generic-error-sent {
+                sensor pppoe-error-statistics {
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/generic-error-sent;
+                }
+                type integer;
+                description "Number of generic-error-sent";
+            }
+            field malformed-packets-rec {
+                sensor pppoe-error-statistics {
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/malformed-packets-received;
+                }
+                type integer;
+                description "Number of malformed-packets-rec";
+            }
+            field service-error-received {
+                sensor pppoe-error-statistics {
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/service-error-received;
+                }
+                type integer;
+                description service-error-received;
+            }
+            field service-error-sent {
+                sensor pppoe-error-statistics {
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/service-error-sent;
+                }
+                type integer;
+                description "Number of service-error-sent";
+            }
+            field unknown-packets-received {
+                sensor pppoe-error-statistics {
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/unknown-packets-received;
+                }
+                type integer;
+                description "Number of unknown-packets-received";
+            }
+            trigger ac-error-received {
+                synopsis "PPPoE ac-error-received kpi";
+                description "Sets health based on the change in ac-error-received count";
+                frequency 90s;
+                term is-error-increasing {
+                    when {
+                        increasing-at-least-by-value "$ac-error-received" {
+                            value 10;
```

Comment:
Can we parameterize the min-increase-value by creating a variable so that the operator can tweak it?

Reply:
Currently no support for variable. We already have enhancement request for variable.

Reply:
Can we prioritize the fix? I don't think that we should be shipping rules with hardcoded values.